DemoCode/XXXX/ProjectAccess/Public/Get-ProjectAccess.ps1

```diff
@@ -5,7 +5,8 @@ function Get-ProjectAccess {
         [ArgumentCompleter({
             param($commandName, $parameterName, $wordToComplete, $commandAst, $fakeBoundParameters)
             $script:Configuration = $script:Configuration ?? $(Get-EnvironmentConfiguration)
-            Invoke-ProjectNameCompleter -WordToComplete $wordToComplete
+
+            & (Get-Module ProjectAccess) { Invoke-ProjectNameCompleter -WordToComplete $wordToComplete }
```

Comment:
Really cool mic drop. But this won't work akshually.

You need to pass the value across module scopes. 
```suggestion
            & (Get-Module ProjectAccess) { param($wc) Invoke-ProjectNameCompleter -WordToComplete $wc } $wordToComplete
```

![mic-drop-drop-all-the-mics](https://github.com/user-attachments/assets/4632978d-9dbd-49c4-970b-716115ed056a)

Reply:
(in reality seem to work in both ways, or simply calling `Invoke-ProjectNameCompleter  -WordToComplete $wordToComplete` because in all cases we are already in the module scope)

Reply:
Good call on the param, another place where $null annoyingly ends up meaning "search all" and leads to a subtle bug like this, thanks.

You do need the module scope however, because argument completers execute in the caller scope, not the module scope.

![image](https://github.com/user-attachments/assets/df6bc9fe-5f99-4d58-90d4-f36f5355bae1)
![image](https://github.com/user-attachments/assets/0e0327e3-a9b8-4e9f-8a2f-90ba06cbeb05)